sms-checker/notification_lambda.py

```diff
@@ -7,7 +7,13 @@ def lambda_handler(event,context) -> None:
     '''Runs the notification pipeline'''
     logger = logging.getLogger(__name__)
     logging.basicConfig(level=logging.INFO)
-
+    subscribers = get_subscribers()
+    email_list = [entry for entry in subscribers if entry['email']]
+    phone_list = [entry for entry in subscribers if entry['phone']]
+    if email_list:
```

Comment:
Wow, actually sends emails / or text messages. This truly is a work of genius, nay, work of the Gods themselves.